style.css

```diff
@@ -0,0 +1,624 @@
+    }
+
+    #section_inner {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+    }
+
+    .inner-content-container-article {
+        flex: 0 1 auto;
+    }
+
+    .main-project-block:nth-child(odd) {
+        background-color: #F3F6F9;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .project-blocks {
+        flex: 0 1 calc(30% - 1em);
+    }
+
+    .image-wrapper {
+        align-self: center;
+        flex-basis: 300px;
+        flex-grow: 0;
+        flex-shrink: 1;
+        margin: 0 40px 0 40px;
+    }
+
+    #variety-span {
+        order: 2;
+    }
+}
+
+/*--------------900px------------------*/
+
+@media screen and (max-width: 900px) {
+    .header-menu-list {
+        display: flex;
+        justify-content: space-evenly;
+        flex-basis: 20em;
+        padding: 0;
+        margin-right: 1em;
+        flex-grow: 0;
+        flex-shrink: 1;
+        font-size: 1em;
+    }
+
+    .header-content_cta > h1 {
+        font-size: 1.8em;
+    }
+
+    .header-content_cta > h2 {
+        font-size: 1em;
+    }
+
+    .header-content_cta {
+        display: flex;
+        flex-direction: column;
+        padding: 10px 20px 300px 20px;
+        align-items: center;
+        text-align: center;
+    }
+
+    #section_inner {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        width: 100%;
+        margin: 0;
+        padding: 20px;
+    }
+
+    .inner-content-container-article {
+        flex: 0 1 calc(40% - 1em);
+    }
+
+    .main-project-block {
+        background-color: #F3F6F9;
+        display: flex;
+        justify-content: space-between;
+        flex-direction: column;
+    }
+
+    .project-blocks {
+        flex: 0 1 calc(40% - 1em);
+    }
+
+    .image-wrapper {
+        align-self: center;
+        flex-basis: 200px;
+        flex-grow: 0;
+        flex-shrink: 1;
+        margin: 0 40px 0 40px;
+    }
+
+    #variety-span {
+        order: 2;
+    }
+}
+
+/*--------------600px------------------*/
+
+@media screen and (max-width: 600px) {
+    .header-menu-list {
+        display: flex;
+        justify-content: space-evenly;
+        flex-basis: 20em;
+        padding: 0;
+        margin-right: 1em;
+        flex-grow: 0;
+        flex-shrink: 1;
+        font-size: 1em;
+    }
+
+    .header-content_cta>h1 {
+        font-size: 1.5em;
+    }
+
+    .header-content_cta>h2 {
+        font-size: 1em;
+    }
+
+    .header-content_cta {
+        display: flex;
+        flex-direction: column;
+        padding: 10px 20px 300px 20px;
+        align-items: center;
+        text-align: center;
+    }
+
+    #section_inner {
+        display: flex;
+        flex-direction: column;
+        align-content: space-around;
+        width: auto;
+        margin: 0;
+        padding: 20px;
+    }
+
+    .inner-content-container-article {
+        flex: 0 1 calc(30% - 1em);
```

Comment:
Snap de flex-basis op zich wel, maar het ziet er wel erg klein uit als ik de pagina smal maak. Dan zou die nog wel iets meer ruimte in mogen nemen wat mij betreft
![Scherm­afbeelding 2023-01-19 om 18 57 57](https://user-images.githubusercontent.com/23332341/213523529-e0e4e292-5464-499a-869f-4898529e991a.jpg)

Reply:
het is aangepast naar flex: 0 1 calc(50% - 1em); Zonder resultaat